{"name": "test_003_panic_and_power_loss", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable", "trace": "pre_conditions = None\n\n    @severity(severity_level.NORMAL)\n    def test_003_panic_and_power_loss(pre_conditions):\n        Report.comment('******************Test Case*****************')\n        # 1. Press pan/priv button for 6-7 seconds (LED will come on flashing)\n        Report.step('Pan/priv button pressed.')\n        # 2. Check the status of pan/priv accumulator by using the command :rrval uservar 0 8\n        Report.step('Pan/priv accumulator status verified.')\n        Report.comment('command \":rrval uservar 0 8\" sent.')\n>       Report.verify(True, True, 'Pan/priv accumulator is disabled.')\n\nXirgo_KB1_Scripts\\Panic\\test_003_panic_and_power_loss.py:30: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nactual = True, expected = True, fail_string = 'Pan/priv accumulator is disabled.'\n\n    @staticmethod\n    def verify(actual, expected, fail_string):\n>       expect(actual == expected, 'Step ' + str(Report.__step_no) + ': ' + fail_string)\nE       TypeError: 'module' object is not callable\n\nLibraries\\report.py:37: TypeError"}, "attachments": [{"name": "stdout", "source": "2c76f020-604b-4cc8-977e-6e3ecff37226-attachment.txt", "type": "text/plain"}], "start": 1637892237361, "stop": 1637892237365, "uuid": "080b9004-9cb9-4184-af9d-9be08b981941", "historyId": "9b73afe4f6d784d4d0e04292f62f2a97", "testCaseId": "ad0db2d4bed47ab470477f94a5860153", "fullName": "Xirgo_KB1_Scripts.Panic.test_003_panic_and_power_loss#test_003_panic_and_power_loss", "labels": [{"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "Xirgo_KB1_Scripts.Panic"}, {"name": "suite", "value": "test_003_panic_and_power_loss"}, {"name": "host", "value": "WX-AMADRID"}, {"name": "thread", "value": "11616-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Xirgo_KB1_Scripts.Panic.test_003_panic_and_power_loss"}]}